README.md

```diff
@@ -1,11 +1,129 @@
+1. OpenSearch-Network-Stack: Use this stack to either use an existing Vpc or create a new Vpc. This stack also creates a new security group to manage access.
+2. OpenSearch-Infra-Stack: Sets up EC2 ASG (installs opensearch and opensearch-dashboards using userdata), cloudwatch logging, load balancer. Check your cluster log in the log group created from your stack in the cloudwatch.
 
-Be sure to:
+## Getting Started
+
+- Requires [NPM](https://docs.npmjs.com/cli/v7/configuring-npm/install) to be installed
+- Install project dependencies using `npm install` from this project directory
+- Configure [aws credentials](https://docs.aws.amazon.com/cdk/latest/guide/getting_started.html#getting_started_prerequisites)
+
+## Deployment
+
+### Required context parameters
+
+In order to deploy both the stacks the user needs to provide a set of required and optional parameters listed below:
+
+| Name                         | Type    | Description                                                                                             |
+|------------------------------|:--------|:--------------------------------------------------------------------------------------------------------|
+| distVersion (required)       | string  | The OpenSearch distribution version (released/un-released) the user wants to deploy                     |
+| securityDisabled (required)  | boolean | Enable or disable security plugin                                                                       |
+| minDistribution (required)   | boolean | Is it an un-released OpenSearch distribution with no plugins                                            |
+| distributionUrl (required)   | string  | OpenSearch tar distribution url                                                                         |
+| dashboardsUrl (required)     | string  | OpenSearch Dashboards tar distribution url                                                              |
+| cpuArch (required)           | string  | CPU platform for EC2, could be either `x64` or `arm64`                                                  |
+| singleNodeCluster (required) | boolean | Set `true` for single-node cluster else `false` for multi-node                                          |
+| vpcId (Optional)             | string  | Re-use existing vpc, provide vpc id                                                                     |
+| securityGroupId (Optional)   | boolean | Re-use existing security group, provide security group id                                               |
+| cidr (Optional)              | string  | User provided CIDR block for new Vpc, default is `10.0.0.0/16`                                          |
+| managerNodeCount (Optional)  | integer | Number of cluster manager nodes, default is 3                                                           |
+| dateNodeCount (Optional)     | integer | Number of data nodes, default is 2                                                                      |
```

Comment:
should have an option for ML node count also? as we have now MLCommons plugin also.

Reply:
Ack!